include/vctr/Containers/VctrBase.h

```diff
@@ -672,6 +696,30 @@ class VctrBase : private Config,
             clear (data(), n);
     }
 
+    /** Sorts all elements in this vector according to the sortingPredicate function. */
```

Comment:
and the sortingPredicate function is what here? :D

Reply:
Ah copy & paste leftover, will fix that